### PR TITLE
Merge login with current code

### PR DESCRIPTION
**Pull request by emilymarkova**

### CODE REVIEW

File: src/pages/Home.tsx
Comment (bigalex2k) [inline src/pages/Home.tsx:1]: I'm pretty sure on line 4, it mentions importing Background from '../Components/Background'. However, Backround is not there.

File: src/assets/login_background.png
Comment (bigalex2k) [inline src/assets/login_background.png:1]: My background image was probably deleted. I don't see it here.

File: src/Components/PageContent.tsx
Comment (bigalex2k) [inline src/Components/PageContent.tsx:1]: The entire inside this file is gone. Here's what's supposed to be here
<>
    <div className="TitleContent">
        A calender specialized in education
    </div>
    <div className="TitleContent2">
        [Insert name] is a calendar that focuses on its 
        <br>
        </br>
        impact on student’s education.
    </div>
    </>

File: src/App.tsx
Comment (bigalex2k) [inline src/App.tsx:1]: I"m pretty sure there was supposed to be an App.css file in the src folder somewhere, but I don't see it.

**Review by bigalex2k — CHANGES_REQUESTED** [review]
1. My App.css file stored in the src file is gone
2. My PageContent.tsx file has had it's inside ripped out. It's currently an empty function.
3. My Background.txs file was deleted and the background picture for the home page was deleted.

I think that's it. I think the code and files for this should be stored within the history of this repository if you need it.

**Comment by emilymarkova** [comment]
1. Not 100% sure but I think the commit might only show what I'm changing so App.css might not be showing up as a file that's part of the commit because I didn't change it.
2. I just took a look at PageContent.tsx, and it seems to be empty at first glance, but if you expand the file (press Edit or View), it seems that the content is still there.
3. ^Same as 1 - I think I just didn't change Background.txs and the background image so that's why they didn't show up on the commit?

**Comment by bigalex2k** [comment]
I think you're right. my bad.

**Review by bigalex2k — APPROVED** [review]

**Pull request merged**